packages/cli-kit/src/node/archiver.integration.test.ts

```diff
@@ -0,0 +1,36 @@
+import {zip} from './archiver.js'
+import {exists, inTemporaryDirectory, mkdir, touch} from '../file.js'
+import {join as joinPath, dirname} from '../path.js'
+import {describe, expect, test} from 'vitest'
+import StreamZip from 'node-stream-zip'
+
+describe('zip', () => {
+  test('zips a directory', async () => {
+    await inTemporaryDirectory(async (tmpDir) => {
+      // Given
+      const zipPath = joinPath(tmpDir, 'output.zip')
+      const outputDirectoryName = 'output'
+      const outputDirectoryPath = joinPath(tmpDir, outputDirectoryName)
+      const structure = [`extensions/first/main.js`, `test.json`]
```

Comment:
Should this test inlcude the `\` for the windows case to test that is is normalized?

Reply:
Note that that's what happens in the implementation does `glob`. We'll get paths that are `C:\\...` that need to be normalized.